src/tests/Interop/common/ComHelpers.h

```diff
@@ -111,14 +132,20 @@ class UnknownImpl
     }
 
 private:
+#ifdef _WIN32
```

Comment:
This can be the same on all platforms, there is no need to special case `_WIN32`.

Reply:
Ok supposing to leave only
`std::atomic<ULONG> _refCount;`
Do we need also to remove the special case for cosntructor and leave only:
`UnknownImpl() : _refCount{ 1 } {};`
in place of:
`#ifdef _WIN32
    UnknownImpl() = default;
#else    
    UnknownImpl() : _refCount{ 1 } {};
#endif    `
?